tests/test_sdk_v3.py

```diff
@@ -0,0 +1,5 @@
+import globus_automate_client
+
+
+def test_import():
+    assert bool(globus_automate_client)
```

Comment:
This is actually kind of a cool test in a sort of reductive way.

Reply:
Haha, very! It was an early best-effort attempt to establish that, at the very least, it was possible to import the SDK. The unit test PR will flesh this out significantly. =)